### ISSUE TITLE
problems with captions (or tooltips) in fullscreen mode - (Issue and fix)

### ISSUE THREAD

**Issue by srikanthpagadala** [xtk/X#95]
# Issue & Fix
#### Issue:

I don't know if you ever noticed that "captions" (or tooltips), sometimes randomly stop appearing. 
In my case, this was more observed when I trigger "fullscreen" of my xtk_container div using HTML5 fullscreen APIs.

More Info about HTML5 fullscreen apis here: 
https://developer.mozilla.org/en-US/docs/DOM/Using_full-screen_mode
http://html5-demos.appspot.com/static/fullscreen.html
#### Fix:

In X/ui/caption.js, line number 194

Before:

// ..render
  this.attach(this._parent);

After:

  // ..render
  //this.attach(this._parent);
  goog.dom.appendChild(this._parent, this.getElement());

Basically, instead of calling "attach" method, simply "append" the caption element to the parent element. This solves the all the caption related problems.
